### PR TITLE
Add tags page styling changes

### PR DESCRIPTION
**Pull request by sudoDeznit**
- Add screen page redesign
- Added floating buttons for "delete" , "tagging", "litter status"
- Added animation to Tagging component now behaves like action sheet
- Added feature -- toggle litter picked up status to individual image.
- Animate and scroll to position of tag when new tags added.
- added translation for newly added components

**Trello Card**
[Change styling of add tags page](https://trello.com/c/HSun8AKD)

### CODE REVIEW

**Comment by xlcrr** [comment]
Can you add grab image to zoom into this PR too please

**Comment by sudoDeznit** [comment]
Pinch and zoom feature is not added in this PR.. will add in a new PR

**Review by xlcrr — APPROVED** [review]

File: screens/pages/AddTags.js [hunk 1]
```diff
@@ -191,246 +199,279 @@ class AddTags extends PureComponent {
...
+                                        />
+                                    )}
+                                    {!this.state.isKeyboardOpen && (
+                                        <TouchableOpacity
+                                            onPress={() => this.addTag()}
+                                            style={styles.buttonStyle}>
+                                            <SubTitle
+                                                color="white"
+                                                dictionary={`${lang}.tag.add-tag`}>
+                                                ADD TAG
```
[the remark this answers is not in the document]
Comment (sudoDeznit) [reply screens/pages/AddTags.js:405]: Which trello board is this ?? It says I have no permission

File: screens/pages/AddTags.js [hunk 2]
```diff
@@ -1,188 +1,196 @@
...
+                this.keyboardStartAnimation(-this.state.keyboardHeight - 400);
+            }
         );
-        this.keyboardDidHideListener = Keyboard.addListener(
+        this.keyboardDidHideSubscription = Keyboard.addListener(
             'keyboardDidHide',
-            this._keyboardDidHide.bind(this)
+            () => {
+                this.setState({ isKeyboardOpen: false, keyboardHeight: 0 });
+                console.log(this.state.keyboardHeight);
```
[the remark this answers is not in the document]
Comment (sudoDeznit) [reply screens/pages/AddTags.js:70]: Will remove

File: screens/pages/components/LitterCategories.js [hunk 3]
```diff
@@ -92,58 +49,50 @@ class LitterCategories extends PureComponent {
...
     card: {
-        marginBottom: 20,
-        borderRadius: 6,
-        paddingTop: 10
-    },
-    container: {
-        height: SCREEN_HEIGHT * 0.125,
-        position: 'absolute',
-        zIndex: 1
+        height: 100,
```
[the remark this answers is not in the document]
Comment (sudoDeznit) [reply screens/pages/components/LitterCategories.js:68]: Even in that case finding a perfect multiplier that works well on all screen sizes is matter of preference... So we can go with this value ... If u find it looks weird in any device size let me know... Will try some other value... But it looked okay to me for most of the devices I tested.

File: screens/pages/AddTags.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sudoDeznit) [reply screens/pages/AddTags.js:405]: Translation is added ... Dictionary prop provides translated value and it  is given precedence over the text.
But I will delete the Text to remove confusion.

File: screens/pages/AddTags.js [hunk 4]
```diff
@@ -191,246 +199,279 @@ class AddTags extends PureComponent {
...
+                            <Pressable
+                                onPress={this.deleteImage}
+                                style={[
+                                    styles.actionButtonStyle,
+                                    { backgroundColor: Colors.error }
+                                ]}>
+                                <Body
+                                    color="white"
+                                    dictionary={`${lang}.tag.delete`}>
+                                    Yes, Delete
```
[the remark this answers is not in the document]
Comment (sudoDeznit) [reply screens/pages/AddTags.js:454]: https://github.com/OpenLitterMap/react-native/pull/142#discussion_r769602542

**Pull request merged**